### ISSUE TITLE
Progress background width property

### ISSUE THREAD

**Issue by etruta** [manumaticx/circularprogress#5]
Possibility to set a specific width to progress background. :)

**Comment by manumaticx** [comment]
like this?

![](https://camo.githubusercontent.com/2f14850508eb93e0225a7709c6c5cfcd6241cc73/68747470733a2f2f7261772e6769746875622e636f6d2f56797368616b682d4b2f54482d50726f6772657373427574746f6e2f6d61737465722f73637265656e73686f74732f70726f677265737373686f74332e706e67)

**Comment by rondog** [comment]
I think adjusting the size of the center layer would adjust the width of the progress background, right?

**Comment by manumaticx** [comment]
yes, but I'm afraid it'll push the progress background circle to the outer edge. So maybe we'll need to add another layer to center the progress background justified with the progress. I'm going to test this asap.

**Comment by etruta** [comment]
@manumaticx yes, like this! Awesome!!
